be/src/storage/rowset/block_split_bloom_filter.cpp

```diff
@@ -46,7 +46,7 @@ void BlockSplitBloomFilter::add_hash(uint64_t hash) {
     // power of 2)
     DCHECK(_num_bytes >= BYTES_PER_BLOCK);
     uint32_t block_size = _num_bytes / BYTES_PER_BLOCK;
```

Comment:
Do you think `num_blocks` is better than `block_size` since it looks like the size of a block?